src/ruby/puppetserver-lib/puppet/server/config.rb

```diff
@@ -6,8 +6,6 @@
 require 'puppet/server/logger'
 require 'puppet/server/jvm_profiler'
 require 'puppet/server/http_client'
-require 'puppet/server/auth_config_loader'
-require 'puppet/server/auth_provider'
```

Comment:
oh jeez, I need more coffee or less wildfires/smoke... I removed the wrong require. Let me fix that.